### PR TITLE
add missing status code 403 for services/create in docs

### LINKED ISSUE #29215: missing status code 403 for `POST /services/create` in api docs
Hi, All, 

I found that in API endpoint `POST /services/create` it is possible to return a status code of 403, however it is not documented.

Please take a look at the following pic:

**Output of `docker version`:**

```
root@ubuntu:~# docker version
Client:
 Version:      1.13.0-rc2
 API version:  1.25
 Go version:   go1.7.3
 Git commit:   1f9b3ef
 Built:        Wed Nov 23 06:24:33 2016
 OS/Arch:      linux/amd64

Server:
 Version:             1.13.0-rc2
 API version:         1.25
 Minimum API version: 1.12
 Go version:          go1.7.3
 Git commit:          1f9b3ef
 Built:               Wed Nov 23 06:24:33 2016
 OS/Arch:             linux/amd64
 Experimental:        false
```

**Output of `docker info`:**

```
root@ubuntu:~# docker info
Containers: 26
 Running: 0
 Paused: 0
 Stopped: 26
Images: 58
Server Version: 1.13.0-rc2
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 367
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: active
 NodeID: mqwom0ky012fjvig86ni43dhm
 Is Manager: true
 ClusterID: qakt3ylxb8hn7kbcoxxo9dzrk
 Managers: 1
 Nodes: 1
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: About an hour
 Node Address: 192.168.59.103
 Manager Addresses:
  192.168.59.103:2377
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 03e5862ec0d8d3b3f750e19fca3ee367e13c090e
runc version: 51371867a01c467f08af739783b8beafc154c4d7
init version: 949e6fa
Security Options:
 apparmor
Kernel Version: 3.19.0-25-generic
Operating System: Ubuntu 14.04.3 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 1.954 GiB
Name: ubuntu
ID: HRBI:EV6M:3NAC:CXZ3:BDS7:UOKL:PD6Y:ZCK4:V3PF:V7MR:CU2Y:3LGG
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 38
 Goroutines: 189
 System Time: 2016-12-06T14:59:55.608122049+08:00
 EventsListeners: 0
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

/cc @vdemeester @thaJeztah

### PR DESCRIPTION
**Pull request by allencloud**
fixes #29215 

**- What I did**
1. add status code 403 for api 1.24 doc;
2. add status code 403 for swagger.yml;
3. add `Networks` field in `POST /services/create` example.

Signed-off-by: allencloud <allen.sun@daocloud.io>

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -7131,6 +7131,10 @@ paths:
             example:
               ID: "ak7w3gjqoa3kuz8xcpnyy0pvl"
               Warning: "unable to pin image doesnotexist:latest to digest: image library/doesnotexist:latest not found"
+        403:
+          description: "network not for services"
```
Comment (thaJeztah) [inline api/swagger.yaml]: should we use the actual response here (minus the name), e.g. `"network is not eligible for services"`?

Comment (allencloud) [inline api/swagger.yaml]: Thanks for your feedback. It is OK for me. Now updated, PTAL

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by vdemeester** [comment]
LGTM 🐮

**Pull request merged**